app/domain/conjur/validate_trusted_proxies.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'command_class'
+
+module Conjur
+
+  ValidateTrustedProxies ||= CommandClass.new(
+    dependencies: {
+      fetch_trusted_proxies:  ::Conjur::FetchTrustedProxies.new,
+      logger:                 Rails.logger
+    },
+    inputs:       %i(account proxy_list)
+  ) do
+
+    def call
+      fetch_trusted_proxies_list
+      validate_proxies
+    end
+
+    private
+
+    def fetch_trusted_proxies_list
+      @trusted_proxies_list ||= @fetch_trusted_proxies.(
+        account: @account
+      )
+    end
+
+    def validate_proxies
+      return if @trusted_proxies_list.blank? || @proxy_list.blank?
```

Comment:
i see that we return if `@proxy_list` is blank. in that case we might perform a redundant call to the DB. how about we move the `@proxy_list` emptiness check to the beginning of the class?

Reply:
Also - i remember @jonahx saying that we should avoid `Rails` methods and use only pure `Ruby` methods. `blank?` is a `Rails` method so it is better to use `@proxy_list.nil? || @proxy_list.empty?`.

@jonahx i can't find a reference as to why we should prefer ruby over rails. Can you elaborate?

Reply:
There are a lot of places in our code which we are using `blank?`